draft-ietf-quic-tls.md

```diff
@@ -778,6 +778,21 @@ the connection that is hosted on stream 1.  This sequence number is not visible
 to QUIC.
 
 
+## Receiving Protected Packets
+
+Once an endpoint successfully receives a packet with a given packet number, it
+MUST discard all packets with higher packet numbers if they cannot be
+successfully unprotected with either the same key, or - if there is a key update
+- the next packet protection key (see {{key-update}}).  Similarly, a packet that
```

Comment:
We could require closing the connection if a packet was already processed, under the theory it's either an implementation error or someone intentionally trying to corrupt the connection?

I believe it's as simple as checking if any packets with larger have been processed when a key change occurs?